### ISSUE TITLE
batch normalization question

### ISSUE THREAD

**Issue by kyeo95** [jaygshah/Binary-Neural-Networks#4]
This code below implement batch normalization
(I'm an amateur in keras. I'm sorry)
Ak = keras.layers.BatchNormalization()

but how do your code implement backbatch normalization?
backward layer process exclude back batch normalization process
